File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/search-notifications_item.ts


import { ISearchNotifications_Item } from './isearch-notifications_item';

export class SearchNotifications_Item implements ISearchNotifications_Item {
  
	public NotificationID: number = 0;
	public NotificationSubject: string = "";
	public NotificationMessage: string = "";
	public NotificationContextType: string = "";
	public NotificationContextTypeID: number = 0;
	public ContextID: number = 0;
	public ModifiedDate: Date = new Date(0);
	public AppUserID: number = 0;
	public Unread: boolean = false;
  
}

File: GTTS/INL.SearchService/Models/SearchNotifications_Item.cs
using System;

namespace INL.SearchService.Models
{
    public class SearchNotifications_Item : ISearchNotifications_Item
    {
        public long NotificationID { get; set; }
        public string NotificationSubject { get; set; }
        public string NotificationMessage { get; set; }
        public string NotificationContextType { get; set; }
        public int NotificationContextTypeID { get; set; }
        public long ContextID { get; set; }
        public DateTime ModifiedDate { get; set; }
        public int AppUserID { get; set; }
        public bool Unread { get; set; }
    }
}
